src/bitcoin-cli.cpp

```diff
@@ -239,16 +240,17 @@ class GetinfoRequestHandler: public BaseRequestHandler
         result.push_back(JSONRPCRequestObj("getnetworkinfo", NullUniValue, ID_NETWORKINFO));
         result.push_back(JSONRPCRequestObj("getblockchaininfo", NullUniValue, ID_BLOCKCHAININFO));
         result.push_back(JSONRPCRequestObj("getwalletinfo", NullUniValue, ID_WALLETINFO));
+        result.push_back(JSONRPCRequestObj("getwalletbalances", NullUniValue, ID_BALANCESINFO));
```

Comment:
Super nit (please feel free to ignore): Would it make sense to use `ID_WALLETBALANCES` in place of `ID_BALANCESINFO` to preserve the `getXXX` -> `ID_XXX` naming scheme?

Reply:
Thanks for the feedback. The naming of `getwalletbalances` may change but will keep this in mind.

Reply:
Thanks -- done in #18574 (please review :))